pages/disability-benefits/apply/dependents.md

```diff
@@ -1,13 +0,0 @@
----
-title: Declaration Of Status Of Dependents
-heading: Declaration of status of dependents
-display_title: Add or remove a dependent
-entryname: 686-dependent-status
-layout: page-react.html
----
-<nav class="va-nav-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
-    <li><a href="/">Home</a></li>
-    <li><a href="/disability/"> Disability benefits </a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-6319ef7105b909c67a28938ec883ccb3R5.

Breadcrumbs confirmed to look okay.